crates/lang/ir/src/ir/attrs.rs

```diff
@@ -441,19 +447,21 @@ where
 /// - If there are duplicate ink! attributes.
 /// - If the first ink! attribute is not matching the expected.
 /// - If there are conflicting ink! attributes.
-pub fn sanitize_attributes<I, C>(
+pub fn sanitize_attributes<I, C, R>(
     parent_span: Span,
     attrs: I,
     is_valid_first: &ir::AttributeArgKind,
     mut is_conflicting_attr: C,
+    conflict_reason: R,
```

Comment:
Not sure why we still need this whole parameter and a new `InkAttributeConflictErr` type at all with our combine approach.